webpack/assets/javascripts/react_app/components/AutoComplete/AutoComplete.js

```diff
@@ -27,7 +27,6 @@ class AutoComplete extends React.Component {
       'handleKeyDown',
     ]);
     this._typeahead = React.createRef();
-    debounceMethods(this, 500, ['handleLoading']);
```

Comment:
Why are you removing the debounce? we don't want the spinner to be displayed if the result comes in fast enough

Reply:
thanks, makes sense.. updated !